src/core/diff.ts

```diff
@@ -125,6 +134,8 @@ export class Diff {
     format: string,
     expires: string | undefined,
   ): Promise<DiffResponse | undefined> {
+    if (!this._config) this._config = await Config.load('../../')
```

Comment:
Can you explain why you load this path ?

Reply:
It's the root of the repository (it needs to target the directory that contains a `package.json`)